src/components/Inputs/EmailInput/EmailInput.md

```diff
@@ -1,5 +1,5 @@
 ```jsx
-import { EmailFormatValidator } from '../../../validators/EmailValidator'
```

Comment:
I actually think this was the main issue as the validator didn't exist (your exporting 'default` on other side so you can't used es6 named import here)